zerver/lib/onboarding.py

```diff
@@ -69,32 +70,35 @@ def send_initial_pms(user: UserProfile) -> None:
             + "\n"
         ).format(help_url=help_url)
 
-    content = (
-        _("Hello, and welcome to Zulip!") + "\n"
-        "\n"
-        + _("This is a private message from me, Welcome Bot.")
-        + " "
-        + _("Here are some tips to get you started:")
-        + "\n"
-        "* " + _("Download our [Desktop and mobile apps]({apps_url})") + "\n"
-        "* "
-        + _("Customize your account and notifications on your [Settings page]({settings_url})")
-        + "\n"
-        "* " + _("Type `?` to check out Zulip's keyboard shortcuts") + "\n"
-        "{organization_setup_text}"
-        "\n" + _("The most important shortcut is `r` to reply.") + "\n"
-        "\n"
-        + _("Practice sending a few messages by replying to this conversation.")
-        + " "
-        + _(
-            "If you're not into keyboards, that's okay too; "
-            "clicking anywhere on this message will also do the trick!"
+    with override_language(user.default_language):
+        content = (
```

Comment:
This commit should be broken down into two commits. One for PM and and one for stream message.

Reply:
Done!